lib/docker/compose/session.rb

```diff
@@ -175,28 +175,20 @@ def  build(*services, force_rm:false, no_cache:false, pull:false)
     # @return [String] output of the command
     # @raise [Error] if command fails
     def run!(*args)
-      project_opts = {}
-      case @file
+      file_args = case @file
+        when 'docker-compose.yml'
+          []
```

Comment:
yup; this way, we omit `--file` when it has the default value. I got tired of constantly matching against `--file=docker-compose.yml` in the specs and decided that terseness is good.

In the future, the gem will always omit options that have default value in order to future-proof the gem. Docker has broken the gem twice by removing flags, but because the gem passes every flag on every invocation, it breaks even if someone isn't using the flag. (This also helps the gem work with older versions of `docker-compose` that don't know about certain options -- not a problem unless you attempt to use unsupported options.)